### DISCUSSION TITLE
Button accept inventation jumps out of the screen

### DISCUSSION THREAD

**Discussion by hydex80** [community/community#29122] [Other Feature Feedback, Questions, & Ideas]
I want to join a repository but in chrome, firefox and safari on a Mac newest os the green button of accept repository jumps out of the screen. Seems like the menu is looping. Even when I disable css the problem is occurring.

**Comment by hydex80** [comment]
here is the example how it occurs
https://user-images.githubusercontent.com/17109128/184085222-f6a514e3-c028-42a3-a8ba-08d2bafc11a5.mp4

**Comment by radar-blip** [comment]
This is happening to a couple other members also, 
If you would like, you can file a bug report here:

https://support.github.com/contact/bug-report
